src/PrestaShopBundle/Resources/config/routing/admin/sell/orders/orders.yml

```diff
@@ -173,12 +173,6 @@ admin_orders_update_invoice_note:
     orderId: \d+
     orderInvoiceId: \d+
 
-admin_search_product_for_order_creation:
```

Comment:
isn't a bc to remove an endpoint?

Reply:
This one is a doublon (the same is present upper) and it's detected in the tests
I guess a bad merge or rebase happened
It should be fixed in another PR I know but I don't want to lose time and travis resources for such a trivial thing to fix